eval/src/vespa/eval/eval/test/param_variants.h

```diff
@@ -0,0 +1,23 @@
+#include "eval_fixture.h"
+#include "tensor_model.hpp"
+
+namespace vespalib::eval::test {
+
+// for testing of optimizers / tensor functions
+// we produce the same param three times:
+// as-is, with float cells, and tagged as mutable.
+EvalFixture::ParamRepo &
+add_variants(EvalFixture::ParamRepo &repo,
+             const vespalib::string &name_base,
+             const Layout &base_layout,
+             const Sequence &seq)
+{
+    auto name_f = name_base + "_float";
+    auto name_m = name_base + "_mutable";
+    repo.add(name_base, spec(base_layout, seq), false);
```

Comment:
after discussion: names should be with '@' prefix for mutable and '_f' for float cells (4 variants)